### ISSUE TITLE
🐛 Accessibility issues in the accessibility addon

### ISSUE THREAD

**Issue by alexasselin008** [gsoft-inc/sg-orbit#1080]
## Describe the bug

Here is the list of all accessibility tests that fails in the accessibility addon tab

- [ ] Illustration default : 
    - Images must have alternate text
- [ ] Button icon link button
    - Elements must only use allowed ARIA attributes
- [ ] Card tabs
    - Certain ARIA roles must contain particular children
- [ ] Illustration default
    - Images must have alternate text
- [ ] Link disabled
    - Elements must have sufficient color contrast
- [ ] Listbox sections
    - Certain ARIA roles must contain particular children
- [ ] Select _all_
    - Elements must have sufficient color contrast
- [ ] Tabs _all_ 
    - Certain ARIA roles must contain particular children
